README.md

```diff
@@ -72,6 +75,7 @@ This code would make most sense as a...
 
 ```text
 Your answer...
+controller method would be the best option to use because the avg_rating attribute needs to be made into a loop within a method.
```

Comment:
Please review our suggested answer.  This question was about displaying the rating, not calculating it.